From b4c970021b869b28c7786bbbe5ce05410c861dd6 Mon Sep 17 00:00:00 2001
From: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
Date: Mon, 6 Nov 2023 04:48:58 -0600
Subject: [PATCH] meta-common/recipes-kernel: Add support for GD55 SPI

Adds support for the Giga Device GD55B01GE 128MB flash to the linux kernel

Signed-off-by: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
---
 drivers/mtd/spi-nor/gigadevice.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mtd/spi-nor/gigadevice.c b/drivers/mtd/spi-nor/gigadevice.c
index 1650131fd79a..31eae8167437 100644
--- a/drivers/mtd/spi-nor/gigadevice.c
+++ b/drivers/mtd/spi-nor/gigadevice.c
@@ -52,6 +52,10 @@ static const struct flash_info gigadevice_parts[] = {
 		.fixups = &gd25q256_fixups },
 	{ "gd25b512", INFO(0xc8471a, 0, 64 * 1024, 512,
 			   SECT_4K | SPI_NOR_QUAD_READ) },
+	{
+	  "gd55b01ge", INFO(0xc8471b, 0, 64 * 1024, 2048,
+			SECT_4K | SPI_NOR_QUAD_READ | SPI_NOR_4B_OPCODES |
+			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
 };

 const struct spi_nor_manufacturer spi_nor_gigadevice = {
--
2.17.1

